<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>1003</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>yulinxx-图像测试</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushBtnTest_15">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>二值化</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>亮度对比度</string>
    </property>
   </widget>
   <widget class="QSlider" name="hSliderA">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>590</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1000</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="hSliderC">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>650</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1000</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-1000 ~ 1000</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-1000 ~ 1000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_9">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>旋转</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>77</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>绿+蓝</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>缩放</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_11">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>顺 90度</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_12">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>腐蚀膨胀</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>模糊</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_14">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>提边</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>原图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_2">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>50</y>
      <width>77</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>灰度</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>红</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>绿</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_5">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>蓝</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>红+绿</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>红+蓝</string>
    </property>
   </widget>
   <widget class="QSlider" name="hSliderB">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>620</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1000</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-1000 ~ 1000</string>
    </property>
   </widget>
   <widget class="QSlider" name="hSliderD">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>680</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1000</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelD">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-1000 ~ 1000</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>710</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-1000 ~ 1000</string>
    </property>
   </widget>
   <widget class="QSlider" name="hSliderE">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>710</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1000</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelTipA">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>570</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTipB">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>610</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTipC">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>640</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTipD">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>670</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTipE">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>700</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>效果叠加</string>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_17">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>USM锐化</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnSave">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>560</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_18">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>绘制轮廓</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>几种滤波</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_20">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>马赛克</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_21">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>浮雕</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_22">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>素描</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_23">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>颜色变换</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_24">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>290</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>油画</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_25">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>报纸</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_26">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>散点图A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_27">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>320</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>挂网</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_28">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>色彩调节</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_29">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>360</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>反色</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_30">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>360</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>镜像</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_31">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>美颜</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_32">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>400</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>自动白平衡</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_33">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>400</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>色温</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_reset">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>560</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>复原</string>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelProcessTip">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>890</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="hSliderResize">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>40</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>1331</width>
      <height>931</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1329</width>
       <height>929</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="labelImg">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="labelResize">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelResizeText">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>缩放大小</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_34">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>激光线</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_35">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>440</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>边缘加强</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_36">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>440</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>抠图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_37">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>叠加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_38">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>480</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>剪切PNG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_39">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>480</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>抠图PNG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_40">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>文字</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_41">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>520</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Gamma</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_42">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>520</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>边缘挂网</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnTest_zero">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>560</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>置零</string>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelF">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>750</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-1000 ~ 1000</string>
    </property>
   </widget>
   <widget class="QSlider" name="hSliderF">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>750</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1000</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelTipF">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>740</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelG">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>800</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-1000 ~ 1000</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTipG">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>790</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelH">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>860</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-1000 ~ 1000</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTipH">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>850</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="hSliderG">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>800</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1000</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="hSliderH">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>860</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1000</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelNameA">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>580</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelNameB">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>610</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelNameC">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>640</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelNameD">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>670</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelNameE">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>700</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelNameF">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>740</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelNameG">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>790</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelNameH">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>840</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnFit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>自适应</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>pushBtnTest_1</tabstop>
  <tabstop>pushBtnTest_2</tabstop>
  <tabstop>pushBtnTest_3</tabstop>
  <tabstop>pushBtnTest_4</tabstop>
  <tabstop>pushBtnTest_5</tabstop>
  <tabstop>pushBtnTest_6</tabstop>
  <tabstop>pushBtnTest_7</tabstop>
  <tabstop>pushBtnTest_8</tabstop>
  <tabstop>pushBtnTest_9</tabstop>
  <tabstop>pushBtnTest_10</tabstop>
  <tabstop>pushBtnTest_11</tabstop>
  <tabstop>pushBtnTest_12</tabstop>
  <tabstop>pushBtnTest_13</tabstop>
  <tabstop>pushBtnTest_14</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
